app/utils/index.ts

```diff
@@ -76,3 +74,9 @@ export function getCategoryColor(category: string): string {
   if (category in categories) return categories[category];
   else return categories.Other;
 }
+
+export const legendLinks = {
```

Comment:
It would be better to keep all source data information in one location. These should be added as new properties to the sources in `togglableLayers` in `~/mapbox-layers.ts`